tests/ISA/ISA.Tests/Utils.fs

```diff
@@ -25,4 +33,13 @@ let mySequenceEqual actual expected message =
       message i e
   | i,Some a,None ->
     failwithf "%s. Sequence actual longer than expected, at pos %i found item %A."
-      message i a
\ No newline at end of file
+      message i a
+
+module Expect = 
+    
```

Comment:
ArcTable has custom equality and should work?

Reply:
Yup I saw that too now.. 😅 
Will check out if this works now out of the box.

Reply:
Ah but there is no info about what is different. The function here tells me where the difference in the table is.